### ISSUE TITLE
Activity bar - upload activity

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy#17285]
Part of https://github.com/galaxyproject/galaxy/issues/17284

### Upload
* [x] sidebar with most common upload features
   * choose local file
   * remote file
   * paste and fetch
   * data library
* [x] Advanced upload features are displayed into the central panel, such as Rule-Builder, Collection Uploader, Composite …
* [x] consider renaming to “get data” or “import data”
* [x] (1) that brings you to an advance upload page, not a popup with a single-page
* [x] consider adding help to the side-panel for things like deferred datasets, drag-and-drop etc
* [x] should workflow import be added here?

**Comment by jmchilton** [comment]
The rule builder is hard - but the problem it is solving is hard. It would be nice if labs could setup complex sample sheet import procedures and then at the end of process or after the first upload. Extract the process so an activity could be extracted that just took the sheet and the inputs and applied the rules. We can "save rules" now but only after the initial steps are complete - it would be nice if the rules could capture the inputs and structure the process at a higher level and activities seem like a good in.

**Comment by davelopez** [comment]
I think all of these are addressed in #21338, so I will close this, and we can create other issues to further refine.

@jmchilton should we create an issue to track your comment about the "rules activity" feature?
